pyproject.toml

```diff
@@ -72,6 +73,16 @@ indent-style = "space"
 skip-magic-trailing-comma = false
 line-ending = "lf"
 
+[tool.mypy]
```

Comment:
These rules were taken from: https://careers.wolt.com/en/blog/tech/professional-grade-mypy-configuration

We can fine tune this as we go.